developers/bindings/thing-xml.md

```diff
@@ -509,17 +509,36 @@ The `representation property` will be defined in the thing type XML:
     </thing-type>
 ```
 
-Note that the `representation property` is normally not listed in the `properties` part of the thing type, as this part contains static properties, that are the same for each thing of this thing type.
-The name of the `representation property` identifies a property that is added to the thing in the thing handler upon successful initialization.
+Note that the `representation property` is normally not listed in the `properties` part of the thing type XML, as this part contains static properties, that are the same for all instances of this thing type.
+The name of the `representation property` identifies a property or configuration parameter that is added to the thing in the thing handler upon successful initialization.
 
 ### Representation Property and Discovery
 
-The representation property is being used to auto-ignore discovery results of devices that already have a corresponding thing.
-This happens if a device is being added manually.
-If the new thing is going online, the auto-ignore service of the inbox checks if the inbox already contains a discovery result of the same type where the value of its `representation property` is identical to the value of the `representation property` of the newly added thing.
-If this is the case, the result in the inbox is automatically set to ignored.
-Note that this result is automatically removed when the manual added thing is eventually removed.
-A new discovery would then automatically find this device again and add it to the inbox properly.
+The representation property is used to auto-ignore discovery results of things that already exist in the system.
+This can happen a) if a thing has been created manually, or b) it has been discovered separately by two mechanisms e.g. by mDNS, and by NetBios, or UPnP.
+If the new thing goes online, the auto-ignore service of the inbox checks if the inbox already contains a discovery result of the same type where the value of its `representation property` is identical to the value of the `representation property` of the newly added thing.
+If this is the case, the thing in the inbox is automatically set to ignored.
+Note that this thing is automatically removed when the manually added thing is eventually removed.
+A new discovery would then automatically find this thing again and add it to the inbox properly.
+
+See also [Implementing a Discovery Service](index.md#representation-property)
+
+When the auto-ignore service checks the `representation property`, it first checks if the thing has a `property` of the same name, and then it checks if the thing has a `configuration parameter` of the same name. If a `configuration parameter` is used, then its respective `parameter` should also be described in the `configuration-description` part of the XML:
```

Comment:
Repeated text?  Suggest having a topic covered only once in the section.  Remove?

Reply:
Actually the `configuration-description` part of the XML is **NOT** covered in the rest of the document (IMHO). But I deleted the example in the `representation-property` section and made a reference to the `configuration-description` section. But somebody (you?? / me??) should probably add such `configuration-description` section too. => What do you suggest?

Reply:
If this has not been covered other places in the article, then please ignore my comment.  Thank you.

Reply:
Ok. Done.